README.md

```diff
@@ -24,6 +24,9 @@ Sharing, suggestions and contributions are always welcome! Please take a look at
 ## MOOCS
 
 - [Quantum Mechanics and Quantum Computation](https://www.edx.org/course/quantum-mechanics-quantum-computation-uc-berkeleyx-cs-191x) - A conceptual introduction to the fundamental  principles of quantum mechanics.
+- [Quantum Information Science I, Part I](https://courses.edx.org/courses/course-v1:MITx+8.370.1x+1T2018/course/) - This course module will help you establish a foundation of knowledge for understanding what quantum computers can do, how they work, and how you can contribute to discovering new things and solving problems in quantum information science and engineering.
```

Comment:
Could you please change the link to this [landing page](https://www.edx.org/course/quantum-information-science-i) as the one you have added requires a login. Also we try to keep the descriptions brief, could you please shorten this description before merge?